validator/src/validation/length.rs

```diff
@@ -12,7 +12,7 @@ use indexmap::{IndexMap, IndexSet};
 /// If you apply it on String, don't forget that the length can be different
 /// from the number of visual characters for Unicode
 #[must_use]
-pub fn validate_length<T: ValidateLength>(
+pub fn validate_length<T: ValidateLength<u64>>(
```

Comment:
Why?

Reply:
The function is kind of useless now that there's the trait I think. Also casting the `min`, `max` and `equal` values to `u64` might lead to issues when you for example cast a negative `i32` to `u64`. So I thought to add a generic, so that the `length` and all the parameters have to be the same type to compare them without runtime errors or weird casting issues.

Reply:
We can remove it then I think?

Reply:
None of the functions will be used in the macro from the validator crate so they can be removed unless they're used anywhere else?

Reply:
Yep they can be removed now that we have traits